types/pusher-trigger.d.ts

```diff
@@ -0,0 +1,14 @@
+export interface PusherTrigger {
+    readonly channel: string;
+    readonly event: string;
+    readonly payload: string;
+}
+
+export interface PusherTriggerResponse {
+    readonly size: number;
+    readonly timeout: number;
+}
+
+export declare class PusherTriggerAPI {
+    pusherTrigger(payload: PusherTrigger): Promise<PusherTriggerResponse>;

```

Comment:
Shouldn't it be like this?
https://github.com/pusher/pusher-http-node/blob/85c8c14d841d12fd41fdeefe0f3a0e9ae64b5595/index.d.ts#L14